troposphere/elasticsearch.py

```diff
@@ -81,8 +81,24 @@ class SnapshotOptions(AWSProperty):
 
 class VPCOptions(AWSProperty):
     props = {
-        "SecurityGroupIds": ([basestring], False),
-        "SubnetIds": ([basestring], False)
+        'SecurityGroupIds': ([basestring], False),
+        'SubnetIds': ([basestring], False)
+    }
+
+
+class MasterUserOptions(AWSProperty):
+    props = {
+        'MasterUserARN': (basestring, False),
+        'MasterUserName': (basestring, False),
+        'MasterUserPassword': (basestring, False),
+    }
+
+
+class AdvancedSecurityOptionsInput(AWSProperty):
+    props = {
+        'Enabled': (boolean, False),
+        'InternalUserDatabaseEnabled': (boolean, False),
+        'MasterUserOptions': (MasterUserOptions, false),
```

Comment:
sorry, typo on the last comment
```suggestion
        'MasterUserOptions': (MasterUserOptions, False),
```